requirements.txt

```diff
@@ -1,4 +1,5 @@
-M2Crypto < 1.0.0
+cryptography < 2.0.0
 nose < 2.0.0
 pep8 < 2.0.0
 pyflakes < 1.0.0
+six < 2.0.0
```

Comment:
Sure, I can alter the `http_cookies` import and use sys version instead of the PY2 constant.